cint/cint.py

```diff
@@ -14,10 +15,17 @@ def calc(v):
 
 class Cint(object):
     def __init__(self, val):
-        if isinstance(val, (int, long)):
+        if isinstance(val, (int, long, float)):
             super(Cint, self).__init__(val)
         elif isinstance(val, ctypes._SimpleCData):
             super(Cint, self).__init__(val.value)
+        elif isinstance(val, str):
```

Comment:
Hm...  since this is a `Cint` class maybe we should have a `Cfloat` class and put this (and other) logic there?

Reply:
Also we could just support `str` as `super(T, self).__init__(val)` since:
```
In [5]: float('-0.0')
Out[5]: -0.0

In [6]: float('NAN')
Out[6]: nan

In [7]: float('NAn')
Out[7]: nan

In [8]: float('nAn')
Out[8]: nan

In [9]: float('-nAn')
Out[9]: nan

In [10]: float('-InF')
Out[10]: -inf
```

and `int('123')` works as well